cmd/cosmos-sdk-cli/cmd/init.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"go/build"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/gobuffalo/packr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(initCmd)
+}
+
+func resolveProjectPath(remoteProjectPath string) string {
+	gopath := os.Getenv("GOPATH")
+	if gopath == "" {
+		gopath = build.Default.GOPATH
+		// Use $HOME/go
+	}
+	return gopath + string(os.PathSeparator) + "src" + string(os.PathSeparator) + remoteProjectPath
+}
+
+var initCmd = &cobra.Command{
+	Use:   "init AwesomeProjectName",
+	Short: "Initialize your new cosmos zone",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("Project name is required")
+		}
+		projectName := args[0]
+		capitalizedProjectName := strings.Title(projectName)
+		shortProjectName := strings.ToLower(projectName)
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Println("Thank you for using cosmos-zone tool.")
+		fmt.Println("You are only a few steps away from creating your brand new blockchain project on Cosmos.")
+		fmt.Print("We will ask you a few more questions to guide you through this process\n\n")
+		fmt.Print("To configure this project we need a remote project path. If you are unsure you can leave this empty. ")
+		fmt.Print("Remote project path is usually something like github.com/your_user_name/project_name\n")
+		fmt.Print("Enter remote project path: ")
```

Comment:
use backticks to define a string with newlines instead of a bunch of `fmt.Print`'s  ?

Reply:
Yeah, this needs to be changed as well.